demo/kubernetes/rook-operator.yaml

```diff
@@ -1,3 +1,59 @@
+# TODO: Update to rbac.authorization.k8s.io/v1beta1 once v1.5 support is dropped
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
Thanks for doing this. We have been testing using the default service account and thus we have been able to access all these resources.